src/components/ListSection/ListSection.jsx

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable react/prop-types */
+import './ListSection.css';
+
+const ListSection = ({ title, children, clickable = false }) => {
+    return (
+        <div className="list-section">
+            {typeof title === 'string' ? (
+                <h2 className="list-section-title">{title}</h2>
+            ) : (
+                <div className="list-section-title">{title}</div>
+            )}
+            <ul className="list-section-children">
+                {children.map((child, index) => {
+                    const key = `${index}-${typeof child === 'object' ? child.label : child}`;
+                    if (clickable && typeof child === 'object') {
```

Comment:
It looks this chane of conditions could be reworked as you check clickable teice in different branches